### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shuklanirdesh82**
- Adding information for leveraging vDVS as managed plugin
- Adding new published blog entry
- Removing information for deb/rpms (stop generating builds from 0.13 release)
- Additional cosmetic changes to existing readme text

Fixes one of the action item #1051

/cc @tusharnt @vmware/cna_storage-maintainers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,11 +62,14 @@ The Docker volume plugin requires the docker engine to be installed as a prerequ
 Ubuntu users to configure the docker repository and pull the `docker-engine` package from there.
 Ubuntu users can find instructions [here](https://docs.docker.com/engine/installation/linux/ubuntulinux/).
 
-[Docker recommends that the docker engine should start after the plugins.] (https://docs.docker.com/engine/extend/plugin_api/)
+[Docker recommends that the docker engine should start after the plugins.](https://docs.docker.com/engine/extend/plugin_api/)
+
+Starting 0.13 release, no more DEB/RPMs as vDVS(vSphere docker volume service) is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description)! It requires to have docker 1.13/17.03 and above.
```
Comment (govint) [inline README.md]: Could reword as
"Starting 0.13 release, DEB/RPMs will not be available as vDVS(vSphere docker volume service) is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description)! The service requires to have docker 1.13/17.03 or later installed.

File: README.md [hunk 2]
```diff
@@ -54,11 +62,14 @@ The Docker volume plugin requires the docker engine to be installed as a prerequ
 Ubuntu users to configure the docker repository and pull the `docker-engine` package from there.
 Ubuntu users can find instructions [here](https://docs.docker.com/engine/installation/linux/ubuntulinux/).
 
-[Docker recommends that the docker engine should start after the plugins.] (https://docs.docker.com/engine/extend/plugin_api/)
+[Docker recommends that the docker engine should start after the plugins.](https://docs.docker.com/engine/extend/plugin_api/)
```
Comment (govint) [inline README.md]: This seems out of place here, can it be moved to a Q/A section below or an FAQ below.

File: README.md (hunk 1 above)
Comment (govint) [inline README.md]: Should this be located later in a separate section, indicating the DEB/RPM packages are deprecated henceforth.

File: README.md [hunk 3]
```diff
@@ -54,11 +62,14 @@ The Docker volume plugin requires the docker engine to be installed as a prerequ
...
+
+Starting 0.13 release, no more DEB/RPMs as vDVS(vSphere docker volume service) is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description)! It requires to have docker 1.13/17.03 and above. 
+
+Install the plugin from Docker store by specifying “–grant-all-permissions” option. This option can be omitted to choose permissions interactively. You can customize the plugin name using “–alias” option.
 
 ```
-sudo dpkg -i <name>.deb # Ubuntu or deb based distros
-sudo rpm -ivh <name>.rpm # Photon or rpm based distros
+docker plugin install --grant-all-permissions --alias vsphere store/vmware/docker-volume-vsphere:0.12
 ```
```
Comment (govint) [inline README.md]: Suggest keeping all plugin installation discussion and steps in one section and ensure no duplication of the same details.

File: README.md [hunk 4]
```diff
@@ -15,14 +17,20 @@ Detailed documentation can be found in our [GitHub Documenation Page](http://vmw
...
 The download consists of 2 parts:
 
-1. ESX: The ESX code is packaged as [a vib or an offline depot] (http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
-2. VM Running Docker: The docker service is packaged as a deb or rpm file.
-   * Photon/RedHat => Download RPM
-   * Ubuntu => Download Deb.
+1. ESX: The ESX code is packaged as [a vib or an offline depot](http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
+2. VM Running Docker: Plugin is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description). 
+
+The plugin could be installed on Docker (1.13+) as follows:
```
Comment (govint) [inline README.md]: Please make this line and the below steps all under an "Installation" section.

File: README.md (hunk 1 above)
Comment (govint) [inline README.md]: Docker version has already been mentioned above.

File: README.md [hunk 5]
```diff
@@ -139,10 +135,10 @@ VMware ESXi:
 - 6.0 u2
 - 6.5
 
-Docker: 1.9 and higher
+Docker: 1.9 and higher (Recommended 1.13/17.03 and above to use managed plugin)
```
Comment (govint) [inline README.md]: Shouldn't this be like, "Docker versions earlier than 1.13 aren't supported any more as the vDVS is only supported as a managed plugin (Docker versions 1.13/17.03 and later).

File: README.md [hunk 6]
```diff
@@ -139,10 +135,10 @@ VMware ESXi:
...
 
 Guest Operating System:
-- [Photon 1.0] (https://vmware.github.io/photon/) (Includes open-vm-tools)
+- [Photon 1.0](https://vmware.github.io/photon/) (Includes open-vm-tools)
```
Comment (govint) [inline README.md]: Pls. include version to avoid the vsocket issue.

File: README.md [hunk 7]
```diff
@@ -2,7 +2,9 @@
 
 # vSphere Docker Volume Service
 
-vSphere Docker Volume Service enables customers to address persistent storage requirements for Docker containers in vSphere environments. This service is integrated with [Docker Volume Plugin framework](https://docs.docker.com/engine/extend/plugins_volume/). Docker users can now consume vSphere Storage (vSAN, VMFS, NFS) to stateful containers using Docker.
+vSphere Docker Volume Service (vDVS) enables customers to address persistent storage requirements for Docker containers in vSphere environments. This service is integrated with [Docker Volume Plugin framework](https://docs.docker.com/engine/extend/plugins_volume/). Docker users can now consume vSphere Storage (vSAN, VMFS, NFS) to stateful containers using Docker.
+
+vDVS is now Docker certified and available on the [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description). More information is found at [VMware's official blog](https://blogs.vmware.com/virtualblocks/2017/03/29/vsphere-docker-volume-service-now-docker-certified/)
```
Comment (pdhamdhere) [inline README.md]: README.md should not point to blog but rather point to [documentation](vmware.github.io/docker-volume-vsphere/documentation/)

File: README.md [hunk 8]
```diff
@@ -15,14 +17,20 @@ Detailed documentation can be found in our [GitHub Documenation Page](http://vmw
 
 ## Download
 
-**[Click here to download (Github releases)] (https://github.com/vmware/docker-volume-vsphere/releases)**
+**Click here to download [Github releases](https://github.com/vmware/docker-volume-vsphere/releases)**
```
Comment (pdhamdhere) [inline README.md]: Original was fine.

Comment (pdhamdhere) [inline README.md]: Aren't we going to publish to bintray?

File: README.md (hunk 4 above)
Comment (pdhamdhere) [inline README.md]: We should retain RPM/Deb in this release. There should be overlap for at least one release.

Download section should not have Install instructions.

File: README.md (hunk 2 above)
Comment (pdhamdhere) [inline README.md]: Yes, please remove this.

File: README.md (hunk 1 above)
Comment (pdhamdhere) [inline README.md]: Please keep RPM/Deb in this release. You can add a note that DEB/RPMs are deprecated going forward and will not be available

For Docker 1.13/17.x, install vDVS managed plugin as follows;
`docker plugin install --grant-all-permissions --alias vsphere store/vmware/docker-volume-vsphere:<release>` 

Remove "no question asked and pretend the plugin name is 'vsphere' "

File: README.md (hunk 2 above)
Comment (shuklanirdesh82) [inline README.md]: It was just spacing/formatting fix.

Ok, removing this.

File: README.md (hunk 8 above)
Comment (shuklanirdesh82) [inline README.md]: original => ``` [Click here to download (Github releases)] (https://github.com/vmware/docker-volume-vsphere/releases)```
with this change => ```Click here to download Github releases```

Just a formatting change, please let me what you think.

Comment (shuklanirdesh82) [inline README.md]: Github release page still shows the change log and release note. Deliverable will be removed and bintray link for VIB download will be provided from Github release page.

what do you suggest?

Comment (pdhamdhere) [inline README.md]: I see. Original was broken with nested parenthesis. Fine with me.

File: README.md (hunk 6 above)
Comment (shuklanirdesh82) [inline README.md]: Done!

File: README.md (hunk 8 above)
Comment (pdhamdhere) [inline README.md]: Makes sense.

File: README.md (hunk 7 above)
Comment (pdhamdhere) [inline README.md]: Change it to comply with publisher agreement;

vDVS is Docker Certified to use with Docker Enterprise Edition and available in [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description). 

Display "Docker Certified" logo somewhere?

File: README.md [hunk 9]
```diff
@@ -159,7 +153,7 @@ Guest Operating System:
 
 ### Public
 * [containers@vmware.com](cna-storage <containers@vmware.com>)
```
Comment (tusharnt) [inline README.md]: should be changed to : [containers@vmware.com](containers@vmware.com)

Comment (shuklanirdesh82) [inline README.md]: https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description is also pointing to the same email id.

BTW, I haven't touched that line; not sure if it is showing into the diff

File: README.md (hunk 7 above)
Comment (shuklanirdesh82) [inline README.md]: Done! Please have look at the latest diff.

File: README.md (hunk 4 above)
Comment (shuklanirdesh82) [inline README.md]: yeah, that's correct! removing from here and will be making sure no repetition.

File: README.md (hunk 3 above)
Comment (shuklanirdesh82) [inline README.md]: Sure1

File: README.md [hunk 10]
```diff
@@ -1,8 +1,12 @@
 [![Build Status](https://ci.vmware.run/api/badges/vmware/docker-volume-vsphere/status.svg)](https://ci.vmware.run/vmware/docker-volume-vsphere) [![Join the chat at https://gitter.im/vmware/docker-volume-vsphere](https://badges.gitter.im/vmware/docker-volume-vsphere.svg)](https://gitter.im/vmware/docker-volume-vsphere?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
+[<img src="https://github.com/vmware/docker-volume-vsphere/blob/docUpdateManagedPlugin.shuklanirdesh82/docs/misc/Docker%20Certified.png" width="96">](https://store.docker.com/plugins/vsphere-docker-volume-service?tab=description)
```
Comment (pdhamdhere) [inline README.md]: Can it be added as badge?

Link is pointing to private branch. It should point to master.

File: README.md [hunk 11]
```diff
@@ -1,8 +1,12 @@
...
+
 # vSphere Docker Volume Service
 
-vSphere Docker Volume Service enables customers to address persistent storage requirements for Docker containers in vSphere environments. This service is integrated with [Docker Volume Plugin framework](https://docs.docker.com/engine/extend/plugins_volume/). Docker users can now consume vSphere Storage (vSAN, VMFS, NFS) to stateful containers using Docker.
+vSphere Docker Volume Service (vDVS) enables customers to address persistent storage requirements for Docker containers in vSphere environments. This service is integrated with [Docker Volume Plugin framework](https://docs.docker.com/engine/extend/plugins_volume/). Docker users can now consume vSphere Storage (vSAN, VMFS, NFS) to stateful containers using Docker.
```
Comment (pdhamdhere) [inline README.md]: Replace https://docs.docker.com/engine/extend/plugins_volume/ with https://docs.docker.com/engine/extend/

File: README.md [hunk 12]
```diff
@@ -15,14 +19,20 @@ Detailed documentation can be found in our [GitHub Documenation Page](http://vmw
 
 ## Download
 
-**[Click here to download (Github releases)] (https://github.com/vmware/docker-volume-vsphere/releases)**
+**Click here to download from [Github releases](https://github.com/vmware/docker-volume-vsphere/releases)**
```
Comment (pdhamdhere) [inline README.md]: +**[Click here to download latest Github release](https://github.com/vmware/docker-volume-vsphere/releases)**

File: README.md (hunk 1 above)
Comment (shuklanirdesh82) [inline README.md]: Done!

File: README.md [hunk 13]
```diff
@@ -54,12 +56,13 @@ The Docker volume plugin requires the docker engine to be installed as a prerequ
 Ubuntu users to configure the docker repository and pull the `docker-engine` package from there.
 Ubuntu users can find instructions [here](https://docs.docker.com/engine/installation/linux/ubuntulinux/).
 
-[Docker recommends that the docker engine should start after the plugins.] (https://docs.docker.com/engine/extend/plugin_api/)
+Starting 0.13 release, DEB/RPMs are deprecated going forward and will not be available, with the vDVS(vSphere docker volume service) being available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description)! It requires to have docker 1.13/17.03 and above.
```
Comment (pdhamdhere) [inline README.md]: ### On Docker Host (VM)
For Docker 1.13 and above, install the plugin from Docker Store.
```
docker plugin install --grant-all-permissions --alias vsphere store/vmware/docker-volume-vsphere:0.13
```
DEB/RPM packages will be deprecated going forward and will not be available.
```
-sudo dpkg -i <name>.deb # Ubuntu or deb based distros
-sudo rpm -ivh <name>.rpm # Photon or rpm based distros
```

File: README.md [hunk 14]
```diff
@@ -139,10 +127,10 @@ VMware ESXi:
 - 6.0 u2
 - 6.5
 
-Docker: 1.9 and higher
+Docker: 1.9 and higher (Recommended 1.13/17.03 and above to use managed plugin)
```
Comment (pdhamdhere) [inline README.md]: Change this to Docker 1.12 and higher

Also move this line at the end of GOS section

File: README.md [hunk 15]
```diff
@@ -139,10 +127,10 @@ VMware ESXi:
 - 6.0 u2
 - 6.5
```
Comment (pdhamdhere) [inline README.md]: Can we just put them as;

VMware ESXi:
- 6.0, 6.0u1, 6.0u2
- 6.5

File: README.md [hunk 16]
```diff
@@ -139,10 +127,10 @@ VMware ESXi:
...
 
-Docker: 1.9 and higher
+Docker: 1.9 and higher (Recommended 1.13/17.03 and above to use managed plugin)
 
 Guest Operating System:
-- [Photon 1.0] (https://vmware.github.io/photon/) (Includes open-vm-tools)
+- [Photon 1.0, Revision 2](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-10-revision-2-binaries) (v4.4.51 or later)
```
Comment (pdhamdhere) [inline README.md]: Put Photon at the bottom of list :(

File: README.md (hunk 9 above)
Comment (pdhamdhere) [inline README.md]: Change to [containers@vmware.com](containers@vmware.com)

File: README.md [hunk 17]
```diff
@@ -159,7 +147,7 @@ Guest Operating System:
 
 ### Public
 * [containers@vmware.com](cna-storage <containers@vmware.com>)
-* [Issues] (https://github.com/vmware/docker-volume-vsphere/issues)
+* [Issues](https://github.com/vmware/docker-volume-vsphere/issues)
```
Comment (pdhamdhere) [inline README.md]: While you are there, can you update " # Known Issues". All of them are already fixed. May be add following-
- Multi-tenancy feature is limited to single ESX #1032

File: README.md (hunk 15 above)
Comment (shuklanirdesh82) [inline README.md]: How about this?

VMware ESXi: 6.0 and above

File: README.md (hunk 16 above)
Comment (shuklanirdesh82) [inline README.md]: Done!

File: README.md (hunk 14 above)
Comment (shuklanirdesh82) [inline README.md]: done!

File: README.md [hunk 18]
```diff
@@ -15,14 +19,12 @@ Detailed documentation can be found in our [GitHub Documenation Page](http://vmw
...
 The download consists of 2 parts:
 
-1. ESX: The ESX code is packaged as [a vib or an offline depot] (http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
-2. VM Running Docker: The docker service is packaged as a deb or rpm file.
-   * Photon/RedHat => Download RPM
-   * Ubuntu => Download Deb.
+1. ESX: The ESX code is packaged as [a vib or an offline depot](http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
+2. VM Running Docker: Plugin is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description).
 
 Pick the latest release and use the same version of ESX and VM release.
```
Comment (tusharnt) [inline README.md]: Pick the latest release and use the same version of ESX and VM release. ->  Please use VIB and managed plugin corresponding to the same release.

File: README.md [hunk 19]
```diff
@@ -15,14 +19,12 @@ Detailed documentation can be found in our [GitHub Documenation Page](http://vmw
...
-**[Click here to download (Github releases)] (https://github.com/vmware/docker-volume-vsphere/releases)**
+**[Click here to download from Github releases](https://github.com/vmware/docker-volume-vsphere/releases)**
 
 The download consists of 2 parts:
 
-1. ESX: The ESX code is packaged as [a vib or an offline depot] (http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
-2. VM Running Docker: The docker service is packaged as a deb or rpm file.
-   * Photon/RedHat => Download RPM
-   * Ubuntu => Download Deb.
+1. ESX: The ESX code is packaged as [a vib or an offline depot](http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
```
Comment (tusharnt) [inline README.md]: ESX -> VIB

File: README.md [hunk 20]
```diff
@@ -15,14 +19,12 @@ Detailed documentation can be found in our [GitHub Documenation Page](http://vmw
...
+2. VM Running Docker: Plugin is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description).
```
Comment (tusharnt) [inline README.md]: VM running docker -> Managed plugin

File: README.md [hunk 21]
```diff
@@ -54,12 +56,13 @@ The Docker volume plugin requires the docker engine to be installed as a prerequ
...
 Ubuntu users can find instructions [here](https://docs.docker.com/engine/installation/linux/ubuntulinux/).
 
-[Docker recommends that the docker engine should start after the plugins.] (https://docs.docker.com/engine/extend/plugin_api/)
+Starting 0.13 release, DEB/RPMs are deprecated going forward and will not be available, with the vDVS(vSphere docker volume service) being available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description)! It requires to have docker 1.13/17.03 and above.
 
-```
-sudo dpkg -i <name>.deb # Ubuntu or deb based distros
-sudo rpm -ivh <name>.rpm # Photon or rpm based distros
-```
+Install the plugin from Docker store by specifying “–grant-all-permissions” option. This option can be omitted to choose permissions interactively. You can customize the plugin name using “–alias” option.
```
Comment (tusharnt) [inline README.md]: Why don't we add a "installation" section here explicitly?
Also, in addition to plugin, we should also provide a command showing how to install the VIB on ESX.

File: README.md [hunk 22]
```diff
@@ -134,19 +122,17 @@ logging config format for content details.
 ## Tested on
```
Comment (tusharnt) [inline README.md]: Rewording suggestion : Tested on -> Supported configuration

File: README.md [hunk 23]
```diff
@@ -54,12 +56,13 @@ The Docker volume plugin requires the docker engine to be installed as a prerequ
 Ubuntu users to configure the docker repository and pull the `docker-engine` package from there.
 Ubuntu users can find instructions [here](https://docs.docker.com/engine/installation/linux/ubuntulinux/).
 
-[Docker recommends that the docker engine should start after the plugins.] (https://docs.docker.com/engine/extend/plugin_api/)
+Starting 0.13 release, DEB/RPMs are deprecated going forward and will not be available, with the vDVS(vSphere docker volume service) being available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description)! It requires to have docker 1.13/17.03 and above.
```
Comment (tusharnt) [inline README.md]: Starting 0.13 release, DEB/RPMs are deprecated going forward and will not be available, with the vDVS(vSphere docker volume service) being available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description)!  ->  Please note that DEB/RPMs will be deprecated starting 0.13 release. Instead please use managed plugin from [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description)!

File: README.md (hunk 22 above)
Comment (pdhamdhere) [inline README.md]: No, let's keep it tested until GA.

File: README.md (hunk 23 above)
Comment (shuklanirdesh82) [inline README.md]: done!

File: README.md (hunk 7 above)
Comment (shuklanirdesh82) [inline README.md]: sure!

File: README.md (hunk 4 above)
Comment (shuklanirdesh82) [inline README.md]: done!

File: README.md (hunk 1 above)
Comment (shuklanirdesh82) [inline README.md]: done!

File: README.md (hunk 10 above)
Comment (shuklanirdesh82) [inline README.md]: Image is uploaded and checked into master; reference has been corrected.

File: README.md (hunk 18 above)
Comment (shuklanirdesh82) [inline README.md]: Done!

File: README.md (hunk 22 above)
Comment (shuklanirdesh82) [inline README.md]: Yeah discussed with @tusharnt offline to drop the comment.

File: README.md (hunk 13 above)
Comment (shuklanirdesh82) [inline README.md]: done!

File: README.md (hunk 17 above)
Comment (shuklanirdesh82) [inline README.md]: Updated!

File: README.md [hunk 24]
```diff
@@ -54,11 +54,19 @@ The Docker volume plugin requires the docker engine to be installed as a prerequ
 Ubuntu users to configure the docker repository and pull the `docker-engine` package from there.
 Ubuntu users can find instructions [here](https://docs.docker.com/engine/installation/linux/ubuntulinux/).
 
-[Docker recommends that the docker engine should start after the plugins.] (https://docs.docker.com/engine/extend/plugin_api/)
+**Note:** DEB/RPMs will be deprecated starting vDVS 0.13 release instead please use managed plugin from [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description)! It requires to have Docker 1.13/17.03 and above.
```
Comment (pdhamdhere) [inline README.md]: Replace entire "### On Docker Host (VM)" section with following;

### On Docker Host (VM)
***For Docker 1.13 and above***, install managed plugin from Docker Store.
```
docker plugin install --grant-all-permissions --alias vsphere store/vmware/docker-volume-vsphere:latest
```
**For Docker 1.12 and earlier**, DEB/RPM packages will be deprecated going forward and will not be available.
```
-sudo dpkg -i <name>.deb # Ubuntu or deb based distros
-sudo rpm -ivh <name>.rpm # Photon or rpm based distros
```

File: README.md [hunk 25]
```diff
@@ -121,7 +111,10 @@ journalctl -fu docker.service # Journalctl/Systemd
  "MaxLogSizeMb": 100,
  "LogPath": "/var/log/docker-volume-vsphere.log"}
 ```
-* **Turning on debug logging**: stop the service and manually run with ``--log_level=debug` flag
+* **Turning on debug logging**: You can change the log level by passing `VDVS_LOG_LEVEL` key to `docker plugin install`, e.g.
+```
+ docker plugin install --grant-all-permissions --alias vsphere store/vmware/docker-volume-vsphere:0.12 VDVS_LOG_LEVEL=debug
```
Comment (pdhamdhere) [inline README.md]: 0.12 => latest

File: README.md (hunk 24 above)
Comment (shuklanirdesh82) [inline README.md]: done!

**Comment by shuklanirdesh82** [comment]
@govint  Thanks for your comments, I have addressed all your supplied comment. Please have a look and share your comment if there is any.

**Pull request merged**